modules/registrar.go

```diff
@@ -107,11 +107,11 @@ func (r *Registrar) BuildModules(ctx registrar.Context) jmodules.Modules {
 
 	authModule := auth.NewModule(r.parser, cdc, db)
 	bankModule := bank.NewModule(r.parser, sources.BankSource, cdc, db)
-	distrModule := distribution.NewModule(ctx.JunoConfig, sources.DistrSource, bankModule, db)
+	distrModule := distribution.NewModule(cdc, ctx.JunoConfig, sources.DistrSource, bankModule, db)
```

Comment:
I think it's much better to keep things coherent with already existing code and put the `cdc` param as the second last, right before the `db` param